app/components/content/talk_to_us_component.html.erb

```diff
@@ -25,6 +25,13 @@
           <%= render ChatComponent.new %>
         </div>
       </div>
+        </div>
+      <div class="inset">
+          <%= render Content::InsetTextComponent.new(
+            header: "Non-UK citizens:",
+            text: "You can call us on <a href=\"tel:+448003892500\">+44 800 389 2500</a>, or use the free live chat service. Calls will be charged at your country's standard rate.",
+            color: "purple-white"
```

Comment:
@emmaachesongray The background of this inset on `/help-and-support` is white not purple? I'm happy for this to be fixed retrospectively if needs be

Reply:
I think we should do this retrospectively and with Jen/Martyn. I'm not sure if Jen might've left it white for a reason like so the there's enough colour contrast?

Reply:
Agree - I'll put a ticket in the backlog for next sprint to review when Jen is back